clusterloader2/testing/load/config.yaml

```diff
@@ -7,8 +7,10 @@
 #Constants
 # Cater for the case where the number of nodes is less than nodes per namespace. See https://github.com/kubernetes/perf-tests/issues/887
 {{$NODES_PER_NAMESPACE := MinInt .Nodes (DefaultParam .NODES_PER_NAMESPACE 100)}}
+{{$LARGE_CLUSTER_NODES := DefaultParam .CL2_LARGE_CLUSTER_NODES 1000}}
```

Comment:
It is not about how big cluster is "large" but some test config marking the size of tested services that could be set arbitrarily.

CL2_HUGE_SERVICES_SIZE ?

Reply:
Also - could you add a comment about allowed values - not larger than .Nodes and .Nodes should be divisibile by it.

We should address those restrictions later.

Reply:
The current logic distincts between a case where there are [100, 1000) nodes and more than a 1000 nodes. In the first case there are multiple namespaces and the number of pods is changed to 2 * `$MIN_PODS_IN_SMALL_CLUSTERS`, in the latter it's exactly one namespace and the number of pods per deployment is different.

I believe the most optimal way of setting this would be indeed to have the `$CL2_HUGE_SERVICES_SIZE` variable, applied only when `$ENABLE_HUGE_SERVICES` is `true`. Then we can change the final `if` block to:
```
{{if $ENABLE_HUGE_SERVICES}}
  {{$schedulerThroughputReplicasPerNamespace = DivideInt .Nodes $HUGE_SERVICES_SIZE}}
  {{$schedulerThroughputPodsPerDeployment = DivideInt .Nodes $schedulerThroughputReplicasPerNamespace}}
  {{$schedulerThroughputNamespaces = 1}}
{{end}}
```
Does this sound good to you?

Reply:
I guess we could name it `HUGE_SERVICES_PODS_PER_DEPLOYMENT` instead of `HUGE_SERVICES_SIZE`

Reply:
Both names are fine, HUGE_SERVICES_SIZE is concise and readable for me.

> Does this sound good to you?
Yes, that was exactly my suggestion.

Reply:
After some consideration - `HUGE_SERVICES_PODS_PER_DEPLOYMENT` makes more sense